src/gateway/__tests__/request.test.js

```diff
@@ -27,7 +27,7 @@ describe("#abortInFlightRequests", () => {
             finally: jest.fn().mockReturnThis(),
         };
         jest.spyOn(MakeRequest, "makeRequest").mockReturnValue(fakeRequest);
-        request(fakeOptions, fakeLogger, "URL");
+        request(fakeLogger, "URL", fakeOptions);
```

Comment:
They are optional now, so they have to go at the end. This is more idiomatic, I think, anyway.

Reply:
Yep! Like this, thank you.